packages/dashboard-frontend/src/containers/IdeLoader.tsx

```diff
@@ -310,7 +310,11 @@ class IdeLoaderContainer extends React.PureComponent<Props, State> {
       this.setState({
         currentStep: LoadIdeSteps.START_WORKSPACE,
       });
-    } else if (workspace?.isStopped && this.state.currentStep !== LoadIdeSteps.INITIALIZING) {
+    } else if (
+      workspace?.isDevWorkspace &&
+      workspace?.isStopped &&
+      this.state.currentStep !== LoadIdeSteps.INITIALIZING
+    ) {
       this.showAlert({
         title: `Workspace "${this.workspaceName}" is failed to start.`,
```

Comment:
```suggestion
        title: `Workspace "${this.workspaceName}" has failed to start.`,
```
Not related to this PR, but this change would be great

Reply:
I have fixed it